versions/1.0.0/Dockerfile

```diff
@@ -0,0 +1,55 @@
+FROM bandsintown/alpine:latest as builder
+
+FROM php:5.6.32-fpm-alpine
+
+ENV S6_OVERLAY_VERSION=1.19.1.1 GODNSMASQ_VERSION=1.0.7 CONSUL_TEMPLATE_VERSION=0.19.4 CONSUL_VERSION=0.8.4 MEMCACHED_DEPS="zlib-dev libmemcached-dev cyrus-sasl-dev" TZ="America/New_York"
+
+COPY --from=builder /usr/local/bin/consul /usr/local/bin/consul
+COPY --from=builder /usr/local/bin/consul-template /usr/local/bin/consul-template
+COPY --from=builder /usr/sbin/go-dnsmasq /usr/sbin/go-dnsmasq
+COPY --from=builder /etc/cont-init.d /etc/cont-init.d
+COPY --from=builder /etc/services.d /etc/services.d
+
+RUN apk update && apk upgrade \
+    && apk add --update \
+      coreutils \
+      freetype-dev \
+      libjpeg-turbo-dev \
+      libltdl \
+      libmcrypt-dev \
+      libpng-dev \
+      curl wget bash tree jq bind-tools su-exec \
+    && apk add --no-cache --update libmemcached-libs zlib \
```

Comment:
Why do you need to run those commands ? They are not defined in your base image ?